tests/nlg/test_conditional_nlg.py

```diff
@@ -41,6 +41,7 @@ def test_generate_utterance_text(nlg_class: ConditionalNLG):
     """
     expected_response1 = AnnotatedUtterance(
         text="something like the A Test Movie Title",
+        utterance_id="CNV1_TEST03_5",
```

Comment:
Not sure it should be addressed in this PR but I find it odd to include the utterance_id of the template in the generated utterance. Maybe the id should not be considered when generating utterance based on templates.

Reply:
The utterance_id is considered in these tests only because it is included in the test data that is used here. I don't think it is a real problem. We are checking here mainly if the generated utterance text is correct.

Reply:
`utterance_id` is set to None while lading utterance templates as we discussed.